spec/views/hq/sponsors/index_spec.rb

```diff
@@ -8,6 +8,19 @@
     expect(view).to render_template partial: 'hq/sponsors/sponsors.html.haml', locals: {sponsors: []}
   end
 
+  it 'shows active & requested sponsorship totals' do
+    assign(:sponsors, [])
+    allow(view).to receive(:total_active_sponsorships).
+      and_return 3
+    allow(view).to receive(:total_requested_sponsorships).
+      and_return 10
+
+    render
+
+    expect(rendered).
+      to have_text 'Currently sponsoring 3 orphans out of 10 requested.'
+  end
+
```

Comment:
I lean towards the opinion I expressed in the scrum: while nothing would change in practical terms by switching to `expect`, I favour the structure that uses `allow` in the setup phase and `expect` in the verification phase. Using `expect` on L13 & 15 would be testing the implementation, while what I'd rather focus on here is the final outcome - that the right text is rendered.